src/menu/settings.rs

```diff
@@ -1,8 +1,23 @@
+use std::fs;
+use std::path::Path;
 use crate::menu::Menu;
 use crate::menu::MenuRoot;
 use crate::prelude::*;
 
+#[derive(Prefs, Reflect, Default)]
+struct Preferences {}
+
 pub(super) fn plugin(app: &mut App) {
+    let config_path = dirs::config_dir().unwrap().join(env!("CARGO_PKG_NAME"));
+
+    if Path::new(&config_path).exists() || fs::create_dir(&config_path).is_ok() {
```

Comment:
```suggestion
    let _ = fs::create_dir_all(&config_path);
    if fs::exists(&config_path) {
```
`fs::create_dir` will only create the directory if it doesn't yet exist, so we can run it unconditionally, ignore the returned `Result`, and only check `fs::exists`. Also, `fs::create_dir_all` will create the full path of parent directories as needed.